birdyboard.py

```diff
@@ -0,0 +1,217 @@
+import os
+import sys
+import csv
+import time
+import uuid
```

Comment:
You're not using `sys`, `csv` or `uuid` in this file anymore, these imports can be removed.